toml/src/main/resources/intentionDescriptions/ExtractDependencySpecificationIntention/after.rs.template

```diff
@@ -0,0 +1,3 @@
+[dependencies.foo]
```

Comment:
That's not true because the intention leaves an empty `[dependencies]` table right now. WDYT about actually removing them?